src/main/java/de/tum/cit/aet/artemis/lecture/web/LectureUnitResource.java

```diff
@@ -207,4 +207,18 @@ public ResponseEntity<LectureUnit> getLectureUnitById(@PathVariable @Valid Long
         lectureUnit.setCompleted(lectureUnit.isCompletedFor(userRepository.getUser()));
         return ResponseEntity.ok(lectureUnit);
     }
+
+    /**
+     * POST lectures/{lectureId}/lecture-units/{lectureUnitId}/ingest
+     * This endpooint is for starting the ingestion of one lecture unit.
+     *
+     * @param lectureUnitId The Id of the Lecture Unit that is <going to be ingested
+     * @return the ResponseEntity with status 200 (OK) and a message success or null if the operation failed
+     */
+    @PostMapping("lectures/{lectureId}/lecture-units/{lectureUnitId}/ingest")
+    public ResponseEntity<Void> ingestLectureUnit(@PathVariable Long lectureUnitId) {
```

Comment:
```suggestion
    @PostMapping("lectures/{lectureId}/lecture-units/{lectureUnitId}/ingest")
    @EnforceAtLeastInstructor
    public ResponseEntity<Void> ingestLectureUnit(@PathVariable Long lectureUnitId) {
```
I think you should add authorization here. Please also verify in the method itself that the user is actually the instructor of that specific lecture unit